File: src/client/classes/api/req_res/update-text-response.ts
'use strict'

import { Text } from '@/classes/datas/text'
import { GkillAPIResponse } from '../gkill-api-response'

export class UpdateTextResponse extends GkillAPIResponse {

    updated_text: Text

    constructor() {
        super()
        this.updated_text = new Text()
    }

}

File: src/server/gkill/api/req_res/update_text_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateTextResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedText *reps.Text `json:"updated_text"`
}
